src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -373,6 +373,8 @@ class LicenceService(
           licenceId.toString(),
           updatedLicence.forename!!,
           updatedLicence.surname!!,
+          licenceEntity.crn!!,
+          licenceEntity.submittedBy?.username!!,
```

Comment:
think `licenceEntity` would have the previous details on? we only update a copy of the licence with the submitter.

 think `username` should be fine?

Reply:
Even the submitter is getting updated:
val username = SecurityContextHolder.getContext().authentication.name
    val submitter = communityOffenderManagerRepository.findByUsernameIgnoreCase(username)
      ?: throw ValidationException("Staff with username $username not found")

Reply:
updated.